src/main/java/mate/lib/exception/PropertyFileReadingException.java

```diff
@@ -0,0 +1,7 @@
+package mate.lib.exception;
+
+public class PropertyFileReadingException extends RuntimeException {
```

Comment:
I don't see, where this exception is being used, same with DriverLoadingException

Reply:
I read properties from properties file in previous project and tried to use it in this one but got exception "can't read properties file". That's why I moved everything to util class but forget to delete custom exceptions